sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/LiveMetricsRestAPIsForClientSDKsRestClient.cs

```diff
@@ -8,14 +8,23 @@
 using System.Diagnostics;
 using System.Text.Json;
 using System.Threading;
+using Azure.Core.Pipeline;
 using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals;
 using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals.Diagnostics;
 using Azure.Monitor.OpenTelemetry.LiveMetrics.Models;
 
 namespace Azure.Monitor.OpenTelemetry.LiveMetrics
 {
-    internal partial class QuickPulseSDKClientAPIsRestClient
+    internal partial class LiveMetricsRestAPIsForClientSDKsRestClient
     {
+        private readonly string _host;
+
+        public LiveMetricsRestAPIsForClientSDKsRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string host, string apiVersion = "2024-04-01-preview")
```

Comment:
Remove apiVersion from parameter.

Reply:
Please revisit to validate if this constructor could be removed.

Reply:
removed apiVersion. 
I don't want to remove the extra constructor. I think it's important to provide the endpoint in the ctor so we can store the endpoint as a readonly field.